6001-7000/LIT6936TarWaAmidBeta.xml

```diff
@@ -21,7 +21,25 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                 </availability>
             </publicationStmt>
             <sourceDesc>
-                <p/>
+                <listWit>
+                    <witness corresp="BNFabb20" xml:id="A"/>
+                    <witness corresp="PetermannIINachtrag24" xml:id="D"/>
+                    <witness corresp="BNFet221" xml:id="E"/>
+                    <witness corresp="FSUor40" xml:id="H"/>
+                    <witness corresp="FSUor134"  xml:id="J"/>
+                    <witness corresp="BLorient814" xml:id="M">London, British Library, BL Oriental 814, which is indicated with M in 
+                        <bibl><ptr target="bm:Hoffmann2021Amid"/></bibl>, but with O in 
+                        <bibl><ptr target="bm:DeratSeignobos2015Moise"/></bibl>.</witness>
+                    <witness corresp="EMDA75" xml:id="N"/>
+                    <witness corresp="EMML21" xml:id="P">ʾAddis ʾAbabā, Manbara Lǝʿul Qəddus Mārqos, EMML 21, which is indicated with P
```

Comment:
`<bibl>` inside witness is used elsewhere as well and so probably correct, however, the zotero links will not be resolved as the entire content is rendered as a link to the manuscript record already.

Reply:
In this case, it might be better to write 'Derat/Seignobos 2015' explicitely and without markup.

Reply:
The citation will be resolved, just not the active link. But if each quoted bibliography item is listed in the bibliography for the record, as it should be, then it is not a problem.

@CarstenHoffmannMarburg note that I have updated the Zotero record which missed the names of the editors, the full title, the place, and the publisher. It is important that all Zotero items are as correct and complete as possible, too, unfortunately we have no possibility of consistent peer review there so everyone must be very careful, thank you.